controllers/toolchaincluster/toolchaincluster_controller_test.go

```diff
@@ -160,6 +207,50 @@ func TestClusterControllerChecks(t *testing.T) {
 	})
 }
 
+func TestGetClusterHealth(t *testing.T) {
+	t.Run("Check health default", func(t *testing.T) {
+		// given
+		stable, sec := newToolchainCluster("stable", "test-namespace", "http://cluster.com", withStatus(clusterReadyCondition()))
+
+		cl := test.NewFakeClient(t, stable, sec)
+		reset := setupCachedClusters(t, cl, stable)
+
+		defer reset()
+		controller, req := prepareReconcile(stable, cl, requeAfter)
+		controller.checkHealth = func(context.Context, *kubeclientset.Clientset) (bool, error) {
+			return true, nil
+		}
+
+		// when
+		recresult, err := controller.Reconcile(context.TODO(), req)
```

Comment:
As I mentioned in the first comment and also in the previous review, please use camelCase
```suggestion
		recResult, err := controller.Reconcile(context.TODO(), req)
```